webcompat/static/js/lib/issues.js

```diff
@@ -395,7 +405,12 @@ issues.MainView = Backbone.View.extend({
   fetchModels: function() {
     var headersBag = {headers: {'Accept': 'application/json'}};
     this.issue.fetch(headersBag).success(_.bind(function() {
-      _.each([this.title, this.metadata, this.body, this.labels,
+      // _.find() will return the object if found (which is truthy),
+      // or undefined if not found (which is falsey)
+      this._isNSFW = _.find(this.issue.get('labels'),
```

Comment:
Might be a good idea to add ``!!`` to turn truthy and falsy values into actual booleans.